src/rpm/headers/header.rs

```diff
@@ -163,7 +163,7 @@ where
             .ok_or_else(|| RPMError::TagNotFound(tag.to_string()))
     }
 
-    #[cfg(feature = "signature-meta")]
+    #[allow(unused)]
```

Comment:
It shouldn't be unused, either it's required in meta or any of the the given concrete signer implementations which should end up in a `cfg(any(..))` annotation or be deleted.

Reply:
The reason it's unused is that the `get_entry_data_as_binary()` function is only used by the signature verification code, which is only enabled by `signature-pgp`, and because we don't currently have any public functions for accessing header data directly (they're all `pub(crate)`).  Probably we *should* have those public functions (as we can't predict every single tag a user may want to be able to read), and then it wouldn't be unused.